staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -110,17 +120,14 @@ func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Inte
 			}
 		}
 
-		// TODO: replace with content type negotiation?
-		includeUninitialized := req.URL.Query().Get("includeUninitialized") == "1"
```

Comment:
can you verify this got populated/converted properly ("1" and "true" set to true, unset or anything else set to false)?